packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/index.js

```diff
@@ -513,7 +475,7 @@ const ApiTokenCreateView = () => {
                                 id: 'Settings.apiTokens.form.type',
                                 defaultMessage: 'Token type',
                               })}
-                              value={tokenTypeValue}
+                              value={values?.type}
```

Comment:
Have you tested what happens if values is nullish? You may need to give a fallback value (`__null_reset_value__`)

Reply:
the type can be or undefined or one of these values ('full-access', 'custom', 'read-only'), there are not other possible values but if it would be null it works like the undefined one

Reply:
if it is null we show a validation error